models/webhook.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2017 Gitea. All rights reserved.
```

Comment:
For other files, the copyright has been changed, so I did the same way but I do not care
I will wait for the review to be sure I should keep it or not.